### PR TITLE
Added Splash Screen

### PR DESCRIPTION
**Pull request by Anm0l-K-v01d**
simulator_screenshot_8D502CE4-80BC-44D9-B6FC-79A9EF78D0B1

### CODE REVIEW

**Comment by Anm0l-K-v01d** [comment]
simulator_screenshot_6CB2A2A1-C7C0-4863-A29B-E00FE30FEBFD
simulator_screenshot_D9AEB659-13C8-4942-877F-EA5BD7EC0ED7

Added new FAB, Integrated Chatwoot

**Comment by kedar09** [comment]
@Anm0l-K-v01d, please resolve merge conflicts.

**Review by kedar09 — APPROVED** [review]

File: apps/anthaathi-commerce-mobile-client/src/App.tsx [hunk 1]
```diff
@@ -32,11 +33,272 @@ const theme = {
...
+                components={[
+                  {
+                    _component: CoreComponentType.Header,
+                    key: '123',
+                    title: 'Something',
+                    leftIcon: 'menu',
+                  },
+                  {
+                    _component: HomePageComponentType.ChatFloatingButton,
+                    key: "13"
```
Comment (kedar09) [inline apps/anthaathi-commerce-mobile-client/src/App.tsx:58]: remove this

File: apps/anthaathi-commerce-mobile-client/src/App.tsx [hunk 2]
```diff
@@ -32,11 +33,272 @@ const theme = {
...
+                      {
+                        title: 'Schedule Delivery',
+                        description: 'Description Schedule Delivery',
+                        image: '',
+                      },
+                    ],
+                  },
+                ]}
+              />
+            </ScrollView>
```
Comment (kedar09) [inline apps/anthaathi-commerce-mobile-client/src/App.tsx:301]: remove this lines

**Review by kedar09 — CHANGES_REQUESTED** [review]

**Review by kedar09 — APPROVED** [review]

**Pull request merged**